app/views/includes/contact.jade

```diff
@@ -0,0 +1,22 @@
+//- CONTACT ::
+section#contact
+    .contact-title.left
+        h4.titulo #{contact.contact_title}
+        span.bold.white
+            p.ct-t1 #{contact.contact_subtitle}
+            p.ct-t2 #{contact.call_to_action}
+    .contact-form
+        form#contactForm.ct-form(action='mailto:info@upplication.com')
+            input#attendant(type='hidden', name='attendant', value='')
+            input#phone(type='hidden', name='phone', value='N/A')
+            input#name.form-data(type='text', name='name', placeholder='#{contact.name_placeholder}')
+            input#emailForm.form-data(type='email', name='email', placeholder='#{contact.email_placeholder}')
+            textarea#comment.form-data(name='comment', placeholder='#{contact.comment_placeholder}')
+            .check
+                input#accept-privacy.checkbox(type='checkbox', name='accept')
+                label.ch-label(for='accept-privacy') !{contact.tos_checkbox}
+                // CAPTCHA goes here if necessary
+            input.submit-btn.bold(type='submit', name='', value='#{contact.send_button}')
+            .form-accept.thinest
+                .tick
+                p. #{contact.thanks_text}
```

Comment:
See this: https://stackoverflow.com/questions/34570348/what-is-the-meaning-of-these-red-bars-in-git-file-difference